pkg/machines/components/networks/networkList.jsx

```diff
@@ -29,6 +29,11 @@ import { CreateNetworkAction } from './createNetworkDialog.jsx';
 const _ = cockpit.gettext;
 
 export class NetworkList extends React.Component {
+    shouldComponentUpdate(nextProps, _) {
+        const storagePools = nextProps.storagePools;
+        return !storagePools.find(pool => !pool.name);
```

Comment:
Is that a copy&paste error? This could be the cause of the "find on undefined"? Should this be nextProps.networks or  something such?

Reply:
Ups. FIxed.